infra/core/src/main/java/com/simprints/core/workers/SimCoroutineWorker.kt

```diff
@@ -61,4 +115,11 @@ abstract class SimCoroutineWorker(
             }
         }
     }
+
+    private companion object {
+        private const val WORKER_FOREGROUND_NOTIFICATION_ID = 2
+        private const val WORKER_FOREGROUND_NOTIFICATION_CHANNEL_ID =
+            "WORKER_FOREGROUND_NOTIFICATION"
+        private const val WORKER_FOREGROUND_NOTIFICATION_CHANNEL_NAME = "Sync in progress"
```

Comment:
Shouldn't this also be a localised string since it is shown in a few places in the system UI?